_drafts/SC1-2-2-audio-captions.md

```diff
@@ -0,0 +1,77 @@
+---
+name: audio elements captions
+description: |
+  Non-streaming `audio` elements must not contain auditory information that is not expressed in captions or a text alternative.
+
+success_criterion:
+- 1.2.1 # Audio-only and Video-only (Prerecorded)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- [Audio output][]
+
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming][] `audio` element [audible on the page][].
```

Comment:
Audio that is a media alternative for text is inapplicable/exempt from the SC. Add this info.

Reply:
We have decided to create an additional rule to tackle this and will add this rule to this as a rule group. Thank you for your feedback.